diff --git a/bootstrap.sh b/bootstrap.sh
index 60b6ece..5674500 100755
--- a/bootstrap.sh
+++ b/bootstrap.sh
@@ -151,7 +151,7 @@ chmod +x bin/antRun
 
 echo ... Building Ant Distribution
 
-"${JAVACMD}" -classpath "${CLASSPATH}" -Dant.home=. $ANT_OPTS org.apache.tools.ant.Main -emacs "$@" bootstrap
+"${JAVACMD}" -classpath "${CLASSPATH}" -Dant.home=. $ANT_OPTS org.apache.tools.ant.Main -emacs "$@" bootstrap -Xnocompact -Xnoinlining
 ret=$?
 if [ $ret != 0 ]; then
   echo ... Failed Building Ant Distribution !
